seatunnel-connectors-v2/connector-maxcompute/src/main/java/org/apache/seatunnel/connectors/seatunnel/maxcompute/sink/MaxcomputeSinkFactory.java

```diff
@@ -52,6 +53,8 @@ public OptionRule optionRule() {
                         MaxcomputeSinkOptions.DATA_SAVE_MODE,
                         MaxcomputeSinkOptions.SAVE_MODE_CREATE_TEMPLATE,
                         MaxcomputeSinkOptions.CUSTOM_SQL,
+                        FormatOptions.DATETIME_FORMAT,
+                        FormatOptions.DATETIME_FORMAT_VALUE,
```

Comment:
```suggestion
                        FormatOptions.DATETIME_FORMAT,
```
Only keep the new format.